cmd/util.go

```diff
@@ -336,7 +329,7 @@ func getVersionFromKubectlOutput(s string) string {
 	serverVersionRe := regexp.MustCompile(`Server Version: v(\d+.\d+)`)
 	subs := serverVersionRe.FindStringSubmatch(s)
 	if len(subs) < 2 {
-		printlnWarn(fmt.Sprintf("Unable to get kubectl version, using default version: %s", defaultKubeVersion))
+		glog.V(1).Info(fmt.Sprintf("Unable to get kubelet version, using default version: %s", defaultKubeVersion))
```

Comment:
Have you checked this goes to stderr? If output is in JSON format going to stdout, it shouldn't have any messages interspersed in it.

Reply:
These errors will only be printed if --v 1 is specified, checked it